de.blobbyvolley.BlobbyVolley2.yaml

```
@@ -0,0 +1,65 @@
+app-id: de.blobbyvolley.BlobbyVolley2
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: blobby
+
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --persist=.blobby
+
+modules:
+  # https://github.com/flathub/org.qbittorrent.qBittorrent/blob/79e4ba75b3f71db7a58f51ade240f4bf9deb93c3/org.qbittorrent.qBittorrent.yaml#L29
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-libraries=system
+      - ./b2 install variant=release link=shared runtime-link=shared cxxflags="$CXXFLAGS" linkflags="$LDFLAGS" -j $FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2
+        sha256: 83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1
+
+  # https://github.com/flathub/org.hedgewars.Hedgewars/blob/18630e00d3f2b2a007f5259d1cf42292ff4aecf4/org.hedgewars.Hedgewars.yaml#L32
+  - name: physfs
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://www.icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
+        sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+    cleanup:
+      - /bin
+
+  - name: blobbyvolley2
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/danielknobe/blobbyvolley2.git
+        commit: c1ddb431fe164a86a86b534c0824dbe96c6c658e
+      - type: patch
+        options: 
+          - '--binary'
+        paths: 
+          - c5329cbc50.patch
+          - b069d193b3.patch
+          - data-dir.patch
+          - data-dir-server.patch
+
+  - name: extra-files
+    buildsystem: simple
+    build-commands:
+      - install -D de.blobbyvolley.BlobbyVolley2.png /app/share/icons/de.blobbyvolley.BlobbyVolley2.png
+      - install -D de.blobbyvolley.BlobbyVolley2.desktop /app/share/applications/de.blobbyvolley.BlobbyVolley2.desktop
+      - install -D de.blobbyvolley.BlobbyVolley2.appdata.xml /app/share/metainfo/de.blobbyvolley.BlobbyVolley2.appdata.xml
```

```suggestion
      - install -D de.blobbyvolley.BlobbyVolley2.metainfo.xml /app/share/metainfo/de.blobbyvolley.BlobbyVolley2.metainfo.xml
```
Rename the appdata file to use .metainfo.xml instead